testing/test_samplers_and_trajectories.py

```diff
@@ -5,94 +5,85 @@
 
 from gfn.containers import Trajectories
 from gfn.containers.replay_buffer import ReplayBuffer
-from gfn.envs import DiscreteEBMEnv, HyperGrid
-from gfn.samplers import (
-    BackwardDiscreteActionsSampler,
-    DiscreteActionsSampler,
-    TrajectoriesSampler,
+from gfn.envs import BoxEnv, DiscreteEBMEnv, HyperGrid
+from gfn.examples.box_utils import (
+    BoxPBEstimator,
+    BoxPBNeuralNet,
+    BoxPFEStimator,
+    BoxPFNeuralNet,
 )
-from gfn.utils import DiscretePBEstimator, DiscretePFEstimator
+from gfn.samplers import ActionsSampler, TrajectoriesSampler
+from gfn.utils import DiscretePBEstimator, DiscretePFEstimator, NeuralNet
 
 
-@pytest.mark.parametrize("env_name", ["HyperGrid", "DiscreteEBM"])
-@pytest.mark.parametrize("height", [4, 5])
+@pytest.mark.parametrize("env_name", ["HyperGrid", "DiscreteEBM", "Box"])
 @pytest.mark.parametrize("preprocessor_name", ["KHot", "OneHot", "Identity"])
+@pytest.mark.parametrize("delta", [0.1, 0.5, 0.8])
 def test_trajectory_sampling(
-    env_name: str,
-    height: int,
-    preprocessor_name: str,
-    human_print=False,
+    env_name: str, preprocessor_name: str, delta: float
 ) -> Trajectories:
-    if human_print:
-        print("---Trying Forward sampling of trajectories---")
     if env_name == "HyperGrid":
-        env = HyperGrid(ndim=2, height=height, preprocessor_name=preprocessor_name)
+        if delta != 0.1:
+            pytest.skip("Useless tests")
```

Comment:
I know it's somewhat obvious but would be helpful to explain what makes these tests useless (or refactor so we never attempt a useless test).